UI/CensusBrowseWnd.cpp

```diff
@@ -168,10 +181,15 @@ void CensusBrowseWnd::CompleteConstruction() {
     { counts_species.insert({entry.second, entry.first}); }
     m_population_counts.clear();
 
+    int totalWorlds = 0;
```

Comment:
name variables like `total_worlds` and `total_population`.

Reply:
please rename consistent with expected variable name style

Reply:
Where do you see `camelCase` variable naming in this code they you were mimicking? I'm just seeing `lower_case_underscore_separated` variables and `UPPER_CASE_CONSTANTS`. Functions and classes are `CamelCase`, but that's distinct from locals.